include/beast/websocket/detail/invokable.hpp

```diff
@@ -125,15 +125,17 @@ class invokable
     void
     emplace(F&& f);
 
-    void
+    bool
     maybe_invoke()
     {
         if(base_)
         {
             auto const basep = base_;
```

Comment:
As written, this ensures that `base_` is null _before_ the function gets called. This prevents recursion / callback problems.